README.md

```diff
@@ -431,7 +468,9 @@ This software is still a work-in-progress.  In particular, not all organisms sto
 
 Some ideas for the software were derived from the [ResFinder][resfinder-git], [PointFinder][pointfinder-git], and [PlasmidFinder][plasmidfinder-git] command-line software, as well as from [ABRicate][abricate].
 
-Phenotype/drug resistance predictions are provided with support from the NARMS/CIPARS Molecular Working Group. 
+Phenotype/drug resistance predictions are provided with support from the NARMS/CIPARS Molecular Working Group.
+
+The Multi-locus sequence typing program is from the [MLST] Github.
```

Comment:
Awesome, thanks for including this here.

Could you also add the citations listed in the MLST website ([citations here](https://github.com/tseemann/mlst#citations)) to the **Citations** section?

That is, adding:

>Seemann T, mlst Github https://github.com/tseemann/mlst

>Jolley KA, Bray JE and Maiden MCJ. Open-access bacterial population genomics: BIGSdb software, the PubMLST.org website and their applications [version 1; peer review: 2 approved]. *Wellcome Open Res* 2018, 3:124 (https://doi.org/10.12688/wellcomeopenres.14826.1) 

Note, the citation for PubMLST I got from here (https://pubmlst.org/about.shtml) in case you were wondering why it's different from the one listed on the MLST GitHub (the paper I quoted is a newer paper, from 2018).

Reply:
Added the citations in d2deb2c

Reply:
Awesome. Thanks.